mlir/lib/Dialect/Rock/Transforms/Regularize.cpp

```diff
@@ -153,7 +153,7 @@ static void annotateGenericOp(linalg::GenericOp lgop) {
   size_t argIdx = -1;
   if (lgop.getInputs().size() == 1) {
```

Comment:
does this function work if lgop.getInputs().size() == 0? (not sure if that can happen)